draft-ietf-mmusic-t140-usage-data-channel.xml

```diff
@@ -480,30 +502,42 @@ Answer:
+            overhead, with the maximum assigned transmission interval of T140blocks
+            from the buffer being 500 ms as long as there is text to send.
+          </t>
+          <t>
+            Buffering can also be used for staying within the maximum character
+            transmission rate (<xref target="sec.sdp.dcsa"/>).
           </t>
           <t>
-            An implementation needs to take the user requirements for smooth flow and low
-            latency in real-time text conversation into consideration when assigning a buffer time.
-            It is RECOMMENDED to use the default transmission interval of 300 milliseconds
-            <xref target="RFC4103"/>, or lower, for T.140 data channels.
+            An implementation needs to take the user requirements for smooth
+            flow and low latency in real-time text conversation into consideration when
+            assigning a transmission interval. It is RECOMMENDED to use the default transmission
+            interval of 300 milliseconds <xref target="RFC4103"/>, for T.140 data channels.
+            Implementers MAY also use lower values, for specific applications requiring low latency,
+            taking the increased overhead in consideration.
           </t>
         </section>
         <section anchor="sec.t140.loss" title="Loss of T140blocks">
           <t>
-            In case of network failure or congestion, T.140 data channels might fail and get torn down.
-            If this happens but the session sustains, it is RECOMMENDED that implementations tries to 
-            reestablish the T.140 data channels. If reestablishment of the T.140 data channel is successful,
-            an implementation MUST evaluate if any T140blocks were lost. Retransmission of already successfully
-            transmitted T140blocks MUST be avoided, and missing text markers <xref target="T140ad1"/> SHOULD
-            be inserted in the received data stream where loss is detected or suspected.
+            In case of network failure or congestion, T.140 data channels might
+            fail and get torn down.  If this happens but the session sustains, it
+            is RECOMMENDED that implementations try to reestablish the T.140
+            data channels. As a T.140 data channel does not provide a mechanism
+            for the receiver to identify retransmitted T140blocks after channel
+            reestablishment, the sender MUST NOT retransmit T140blocks unless
+            it has strong indications that a T140block has been lost. Similarly,
+            as a T.140 data channel does not provide a mechanism for a receiver to
+            detect lost T140blocks during channel reestablishment, it MUST NOT insert
+            missing text markers <xref target="T140ad1"/> unless it has reasons to
+            suspect that a T140block might have been lost. Different mechanisms used
+            by senders and receivers to detect or suspect packet loss are outside the
```

Comment:
change "packet loss" to "text loss" because it is resulting text loss after lower level recovery we talk about here.

Reply:
Ok.